arwendebug/errors.go

```diff
@@ -0,0 +1,45 @@
+package arwendebug
+
+import (
+	"errors"
+	"fmt"
+)
+
+// RequestError signals an error
+type RequestError struct {
+	Message  string
+	InnerErr error
```

Comment:
I like this pattern, I was suggesting something like this instead of what we use in elrond-go a long time ago. Used right, an error can tell you the exact trace 👍